### PR TITLE
adds `convox rack ps` and `convox rack logs`

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;0ZXq5ZxK&#x2F;475-convox-rack-ps-and-logs"> convox rack ps and logs</a>

File: cmd/convox/logs.go [hunk 1]
```diff
@@ -20,15 +20,15 @@ func init() {
...
 			},
 			cli.BoolTFlag{
 				Name:  "follow",
-				Usage: "Follow log output (default).",
+				Usage: "keep streaming new log output (default)",
 			},
 			cli.DurationFlag{
 				Name:  "since",
-				Usage: "Show logs since a duration, e.g. 10m or 1h2m10s.",
+				Usage: "Show logs since a duration (e.g. 10m or 1h2m10s)",
```
Comment (mwarkentin) [inline cmd/convox/logs.go]: Minor niggle, but mix of capitalization for usage docs on these various flags.

File: cmd/convox/logs.go [hunk 2]
```diff
@@ -20,15 +20,15 @@ func init() {
 			rackFlag,
 			cli.StringFlag{
 				Name:  "filter",
-				Usage: "Only return logs that match a filter pattern. If not specified, return all logs.",
+				Usage: "filter the logs by a given token",
 			},
 			cli.BoolTFlag{
 				Name:  "follow",
-				Usage: "Follow log output (default).",
+				Usage: "keep streaming new log output (default)",
```
Comment (skwp) [inline cmd/convox/logs.go]: small nitpick but some of the usages start with caps and others don't. the tool would feel more polished with consistency

**Comment by MiguelMoll** [comment]
Ran into this bug where stats for the rack are a no show:

```
⟩ convox rack ps
ID            NAME     RELEASE         STARTED      COMMAND
bec9b9203f25  monitor  20160810023912  9 hours ago  /go/bin/init sh -c api/bin/monitor
b3b3a804b8e7  web      20160810023912  9 hours ago  /go/bin/init sh -c api/bin/web
d87a116fa092  web      20160810023912  9 hours ago  /go/bin/init sh -c api/bin/web

⟩ convox rack ps --stats
ID  NAME  RELEASE  CPU %  MEM  MEM %  STARTED  COMMAND
```

But the original way shows it empty too:

```
⟩ convox ps --app devrack --stats
ID  NAME  RELEASE  CPU %  MEM  MEM %  STARTED  COMMAND
```
Most likely a preexisting issue.

**Pull request closed without merging**